README.md

```diff
@@ -35,6 +35,8 @@ Options:
 --deleteTableMs <ms>  Amount of time tables stay in DELETING state (default: 500)
 --updateTableMs <ms>  Amount of time tables stay in UPDATING state (default: 500)
 --maxItemSizeKb <kb>  Maximum item size (default: 400)
+--ttlCheckEvery <kb>  Time gap between TTL expiration background job (default: 60)
```

Comment:
`<s>`, not `<kb>`, yes?

Reply:
Fixed in https://github.com/architect/dynalite/pull/144/commits/f1f0352eb0f3e91dd9e89e0b7f4e94ca3826a9fd